lib/tes.js

```diff
@@ -33,16 +39,21 @@ class TES {
 
         this.baseURL = config.listener.baseURL;
         this.whSecret = config.listener.secret;
+        
         this.port = config.listener.port || process.env.PORT || 8080;
+
         const serverConfig = {
             ignoreDuplicateMessages: config.listener.ignoreDuplicateMessages === false ? false : true,
-            ignoreOldMessages: config.listener.ignoreOldMessages === false ? false : true
+            ignoreOldMessages: config.listener.ignoreOldMessages === false ? false : true,
+            path : config.path
         }
         this.whserver = whserver(config.listener.server, this.whSecret, serverConfig);
         this._whserverlistener = config.listener.server ? null : this.whserver.listen(this.port);
 
         config.options.debug && logger.setLevel('debug');
         config.options.logging === false && logger.setLevel('none');
+
+        this.config = config; // save options
```

Comment:
is there a benefit to saving the config like this?  It isn't used outside the constructor